app/models/reservation_frame.rb

```diff
@@ -10,4 +10,9 @@ class ReservationFrame < ApplicationRecord
   def is_reserved?
     reservation.present?
   end
+
+  def canceled?
+    self.canceled_at.present?
```

Comment:
ActiveRecord が `{カラム名}?` という名前の predicate method を自動定義してくれます。

```suggestion
    canceled_at?
```

Reply:
そんなに省略しちゃえるんですね。
ありがとうございます！